include/DoocsBackendImageRegisterAccessor.h

```diff
@@ -0,0 +1,142 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+
+#include "DoocsBackendRegisterAccessor.h"
+#include <eq_client.h>
+#include <type_traits>
+
+#include <ChimeraTK/MappedImage.h>
+
+namespace ChimeraTK {
+
+  /// implements conversion from DOOCS image -> MappedImage
+  struct MappedDoocsImgIn : public MappedImage {
```

Comment:
Please swap the two class/struct definitions, so the more important one comes first.